core/tee/tee_svc_storage.c

```diff
@@ -626,6 +627,9 @@ TEE_Result syscall_storage_obj_create(unsigned long storage_id, void *object_id,
 		goto err;
 	}
 
+	/* remove the file if it exists*/
+	if (!filedoesnotexist)
+		tee_file_ops.unlink(file);
```

Comment:
No, we can only assume that each operation is atomic (rename, delete, write...) but not a combination of operations. That is my plan at least. If we need more (i.e., a transaction-based API covering multiple operations), then it will be more complex. Easy on the REE side if we use SQLite. But what about the RPMB FS?

What are you trying to achieve with access/open/unlink/rename that open() could not do with the suitable flags?